setup.cfg

```diff
@@ -0,0 +1,28 @@
+[metadata]
+name = visionmetrics
+version = 0.0.1
+description = Evaluation metric codes for various vision tasks.
+long_description = file: README.md
+long_description_content_type = text/markdown
+author = Ping Jin, Shohei Ono
+license = MIT
+url = https://github.com/microsoft/vision-evaluation
+
+[options]
+packages = find_namespace:
+python_requires = >= 3.8
+install_requires =
+    numpy
+    opencv-python-headless
+    pillow
+    pycocoevalcap
```

Comment:
Remove unused dependencies. If pycocoevalcap is needed, we might want to change it to "extra" dependency. It contains a java binary which we cannot use on production server.